sonar-project.properties

```diff
@@ -1,8 +1,8 @@
 sonar.projectKey=maryiafrantsava-binary_dream_team_belarus
 sonar.organization=maryiafrantsava-binary
 sonar.sources=.
-sonar.exclusions=**/__tests__/**
-sonar.tests=src/__tests__
+# sonar.exclusions=**/__tests__/**
```

Comment:
no need to change this file, we'll add some tests to satisfy sonarcloud later. please undo the changes🙂